src/tests/testHttpRange.cc

```diff
@@ -126,26 +151,13 @@ testRangeCanonization()
 }
 
 int
-main(int, char **)
+main(int argc, char **argv)
 {
-    try {
-        Mem::Init();
-        /* enable for debugging to console */
-        // Debug::debugOptions = xstrdup("ALL,1 64,9");
-        // Debug::BanCacheLogUse();
-        testRangeParser("bytes=0-3");
-        testRangeParser("bytes=-3");
-        testRangeParser("bytes=1-");
-        testRangeParser("bytes=0-3, 1-, -2");
-        testRangeIter();
-        testRangeCanonization();
-    } catch (const std::exception &e) {
-        printf("Error: dying from an unhandled exception: %s\n", e.what());
-        return EXIT_FAILURE;
-    } catch (...) {
-        printf("Error: dying from an unhandled exception.\n");
-        return EXIT_FAILURE;
-    }
-    return EXIT_SUCCESS;
+    Mem::Init();
```

Comment:
If this call is needed or you do not want to check whether it is needed, please move this call to MyTestProgram::startup() where such actions should take place after TestProgram() conversion. See existing implementations of that method for examples. Otherwise, please drop it.

Reply:
Not needed. Removing